packages/scss/demo/templates/form.html

```diff
@@ -10,51 +10,51 @@
 		<link rel="stylesheet" href="../demo.css">
 	</head>
 	<body>
-		<div class="global u-paddingTopBiggest">
-			<div class="container mod-sm">
-				<div class="card">
-					<form class="form">
-						<div class="form-group">
-							<div class="input is-filled">
-								<input class="input-field" type="text" value="input value">
-								<label class="input-label">Input label</label>
+		<main class="main mod-noAside u-paddingTopBiggest">
```

Comment:
That could be a good idea.
As the layout can contain various major elements, I'm not sure we'll be able to do it, but we can have a try.